src/test/apprunner/explorer/apprunnerNode.test.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as FakeTimers from '@sinonjs/fake-timers'
+import * as sinon from 'sinon'
+import { AppRunner } from 'aws-sdk'
+import { verify, anything, instance, mock, when } from 'ts-mockito'
+import { AppRunnerNode } from '../../../apprunner/explorer/apprunnerNode'
+import { AppRunnerServiceNode } from '../../../apprunner/explorer/apprunnerServiceNode'
+import { AppRunnerClient } from '../../../shared/clients/apprunnerClient'
+import { PlaceholderNode } from '../../../shared/treeview/nodes/placeholderNode'
+import { ext } from '../../../shared/extensionGlobals'
+
+describe('AppRunnerNode', function () {
+    let mockApprunnerClient: AppRunnerClient
+    let node: AppRunnerNode
+    let clock: FakeTimers.InstalledClock
+
+    const exampleSummaries: AppRunner.ServiceSummaryList = [
+        {
+            ServiceName: 'test1',
+            Status: 'RUNNING',
+            ServiceArn: 'test-arn1',
+        },
+    ]
+
+    before(function () {
+        clock = FakeTimers.install()
+        ext.awsExplorer = {
```

Comment:
Sandbox this so it doesn't affect other tests

Reply:
Sure. Since this is a new global it should eventually be added to the test set-up. Though right now not much point since it'd just be an empty object. If we're wanting to commit with `mockito` then we can automatically mock and instantiate whatever is needed in the global test set-up.